app/[locale]/(non-interactive)/feedback/page.tsx

```diff
@@ -0,0 +1,342 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import NextButton from "@/components/ui/nextButton";
+import Image from "next/image";
+import { useRouter } from "next/navigation";
+import { FormEvent, useState } from "react";
+import "./styles.css";
+import SubmittedPage from "./submittedPage";
+
+type Props = {};
+
+const Page = (props: Props) => {
+  const [rating, setRating] = useState(0);
+  const [comment, setComment] = useState("");
+  const [submitted, setSubmitted] = useState(false);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const isEmpty = rating === 0 && comment.trim() === "";
+  const router = useRouter();
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (isSubmitting) return;
+    if (isEmpty) {
+      router.push("/end");
+      return;
+    }
+
+    setIsSubmitting(true);
+
+    fetch(`${process.env.NEXT_PUBLIC_WEB_URL}/api/feedback`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        rating,
+        comment,
+      }),
+    });
+
+    setSubmitted(true);
+    setIsSubmitting(false);
+  };
+  if (submitted) {
+    return <SubmittedPage />;
+  }
+  return (
+    <>
+      <Image
+        src="/img/feedback.webp"
+        alt="Description"
+        layout="fill"
+        objectFit="cover"
+        className="fixed"
+      />
+      <form
+        action={process.env.NEXT_PUBLIC_WEB_URL + "/feedback"}
+        method="post"
+        onSubmit={handleSubmit}
+      >
+        <div className="mx-auto flex h-screen w-[390px]  flex-col items-center justify-start">
+          <div className="z-10 mt-[130px] text-center">
+            <p className="text-base text-woh-white">
+              บอกความรู้สึกของคุณที่มีต่อกิจกรรมนี้สิ
+            </p>
+          </div>
+          <div className="z-10 mt-6 flex w-full justify-between px-14">
+            <Button
+              size="sm"
+              id="rating-1"
+              className={`h-[32px] w-[40px] rounded p-0 hover:bg-woh-white ${
+                rating === 1 || rating === 0 ? "bg-woh-white" : "bg-gray"
+              }`}
+              onClick={(e) => {
+                e.preventDefault();
+                setRating((prev) => (prev === 1 ? 0 : 1));
+              }}
+            >
+              <svg
+                width="25"
+                height="25"
+                viewBox="0 0 25 25"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M12.5132 22.2121C18.1777 22.2121 22.7697 17.8706 22.7697 12.5151C22.7697 7.15966 18.1777 2.81818 12.5132 2.81818C6.84879 2.81818 2.25684 7.15966 2.25684 12.5151C2.25684 17.8706 6.84879 22.2121 12.5132 22.2121Z"
+                  stroke={rating === 1 || rating === 0 ? "#C41C11" : "#C5C5C5"}
+                  strokeWidth="2"
```

Comment:
Button 1 -> 5 looks like it can be done in a loop

Reply:
พวก attribute value ใน svg ของแต่ละ btn มันไม่เหมือนกันอะครับ เลย hardcode ไปเลย